test/gtest/uct/ib/test_rc.cc

```diff
@@ -332,15 +332,20 @@ uct_rc_mlx5_iface_common_t* test_rc_mp_xrq::rc_mlx5_iface(entity &e)
     return ucs_derived_of(e.iface(), uct_rc_mlx5_iface_common_t);
 }
 
-void test_rc_mp_xrq::init()
+void test_rc_mp_xrq::set_env_var_or_skip(void *config, const char *var,
+                                         const char *val)
 {
-    ucs_status_t status1 = uct_config_modify(m_iface_config,
-                                             "RC_TM_MP_NUM_STRIDES", "8");
-    ucs_status_t status2 = uct_config_modify(m_iface_config,
-                                             "RC_TM_ENABLE", "y");
-    if ((status1 != UCS_OK) || (status2 != UCS_OK)) {
-        UCS_TEST_SKIP_R("No MP XRQ support");
+    ucs_status_t status = uct_config_modify(config, var, val);
+    if (status != UCS_OK) {
+        UCS_TEST_SKIP_R(std::string("Can't set ") + var);
     }
+}
+
+void test_rc_mp_xrq::init()
+{
+    set_env_var_or_skip(m_iface_config, "RC_TM_MP_NUM_STRIDES", "8");
+    set_env_var_or_skip(m_iface_config, "RC_TM_ENABLE", "y");
+    set_env_var_or_skip(m_md_config, "MLX5_DEVX_OBJECTS", "dct,dcsrq,rcsrq,rcqp");
```

Comment:
shall we add rcsrq,rcqp to DEVX_OJECTS by default?
@Artemy-Mellanox

Reply:
unhandled

Reply:
imo, we can leave it for now. Can introduce `auto` later - when all parts are ready
Waiting for @Artemy-Mellanox comment though

Reply:
ok